shell/debug.sh

```diff
@@ -9,7 +9,11 @@ source "$DIR/lib/bootstrap.sh"
 
 # PACKAGE_TO_DEBUG is the package to debug. If not set, it will default to the
 # main package, which is cmd/<app_name>.
-PACKAGE_TO_DEBUG="${PACKAGE_TO_DEBUG:-$(get_repo_directory)/cmd/$(get_app_name)}"
+if [[ -z $PACKAGE_TO_DEBUG ]]; then
```

Comment:
This is `if not set`. So, we don't need to do `${PACKAGE_TO_DEBUG:-` to default it. Right?

Reply:
Also, wouldn't we want to default to `DEV_CONTAINER_EXECUTABLE` if not set?

We should also update the above comment.